tests/Oryx.Integration.Tests/PythonEndToEndTests.cs

```diff
@@ -3,61 +3,81 @@
 // Licensed under the MIT license.
 // --------------------------------------------------------------------------------------------
 
+using Microsoft.Oryx.BuildScriptGenerator.Python;
+using Microsoft.Oryx.Common;
+using Microsoft.Oryx.Tests.Common;
 using System;
 using System.Collections.Generic;
 using System.IO;
 using System.Threading.Tasks;
-using Microsoft.Oryx.Tests.Common;
 using Xunit;
 using Xunit.Abstractions;
 
 namespace Microsoft.Oryx.Integration.Tests
 {
-    [Trait("category", "python")]
-    public class PythonEndToEndTests : PlatformEndToEndTestsBase
+    public abstract class PythonEndToEndTestsBase : PlatformEndToEndTestsBase
     {
-        private const int ContainerPort = 3000;
-        private const string DefaultStartupFilePath = "./run.sh";
+        protected const int ContainerPort = 3000;
+        protected const string DefaultStartupFilePath = "./run.sh";
 
-        private readonly ITestOutputHelper _output;
-        private readonly string _hostSamplesDir;
-        private readonly string _tempRootDir;
+        protected readonly ITestOutputHelper _output;
+        protected readonly string _hostSamplesDir;
+        protected readonly string _tempRootDir;
 
-        public PythonEndToEndTests(ITestOutputHelper output, TestTempDirTestFixture testTempDirTestFixture)
+        public PythonEndToEndTestsBase(ITestOutputHelper output, TestTempDirTestFixture testTempDirTestFixture)
         {
             _output = output;
             _hostSamplesDir = Path.Combine(Directory.GetCurrentDirectory(), "SampleApps");
             _tempRootDir = testTempDirTestFixture.RootDirPath;
         }
+    }
+
+    [Trait("category", "python")]
+    public class PythonEndToEndTests_BackCompat : PythonEndToEndTestsBase
```

Comment:
@gpcastro you would be interested in tests of this class

Reply:
Let's not use underscores in class names